### PR TITLE
Adding parser for IOSXE/show wireless mobility summary

### LINKED ISSUE #55: 30. show wireless mobility summary
```
ewlc1# show wireless mobility summary
Mobility Summary

Wireless Management VLAN: 299
Wireless Management IP Address: 10.10.7.177
Wireless Management IPv6 Address: 
Mobility Control Message DSCP Value: 48
Mobility Keepalive Interval/Count: 10/3
Mobility Group Name: b80-mobility
Mobility Multicast Ipv4 address: 0.0.0.0
Mobility Multicast Ipv6 address: ::
Mobility MAC Address: 58bf.ea35.b60b
Mobility Domain Identifier: 0x61b3

Controllers configured in the Mobility Domain:

 IP                                        Public Ip                                  MAC Address         Group Name                       Multicast IPv4    Multicast IPv6                              Status                       PMTU 
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
10.10.7.177                               N/A                                        58bf.ea35.b60b      b80-mobility                0.0.0.0           ::                                          N/A                          N/A

```
```
wlc3# show wireless mobility summary
Mobility Summary

Wireless Management VLAN: 299
Wireless Management IP Address: 10.10.7.177
Wireless Management IPv6 Address: 
Mobility Control Message DSCP Value: 48
Mobility Keepalive Interval/Count: 10/3
Mobility Group Name: b80-mobility
Mobility Multicast Ipv4 address: 0.0.0.0
Mobility Multicast Ipv6 address: ::
Mobility MAC Address: 58bf.ea35.b60b
Mobility Domain Identifier: 0x61b3

Controllers configured in the Mobility Domain:

 IP                                        Public Ip                                  MAC Address         Group Name                       Multicast IPv4    Multicast IPv6                              Status                       PMTU 
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
10.10.7.177                               N/A                                        58bf.ea35.b60b      b80-mobility                0.0.0.0           ::                                          N/A                          N/A

```
```
ewlc2# show wireless mobility summary 
Mobility Summary

Wireless Management VLAN: 274
Wireless Management IP Address: 10.10.5.133
Wireless Management IPv6 Address: 
Mobility Control Message DSCP Value: 48
Mobility Keepalive Interval/Count: 10/3
Mobility Group Name: default
Mobility Multicast Ipv4 address: 0.0.0.0
Mobility Multicast Ipv6 address: ::
Mobility MAC Address: 58bf.ea9b.23cb
Mobility Domain Identifier: 0x34ac

Controllers configured in the Mobility Domain:

 IP                                        Public Ip                                  MAC Address         Group Name                       Multicast IPv4    Multicast IPv6                              Status                       PMTU 
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
10.10.5.133                              N/A                                        58bf.ea9b.23cb      default                          0.0.0.0           ::                                          N/A                          N/A

```
```
ewlc3# show wireless mobility summary 
Mobility Summary

Wireless Management VLAN: 10
Wireless Management IP Address: 10.10.30.4
Wireless Management IPv6 Address: 
Mobility Control Message DSCP Value: 48
Mobility Keepalive Interval/Count: 10/3
Mobility Group Name: a0_fabA
Mobility Multicast Ipv4 address: 0.0.0.0
Mobility Multicast Ipv6 address: ::
Mobility MAC Address: 58bf.eacc.0c0b
Mobility Domain Identifier: 0xc82b

Controllers configured in the Mobility Domain:

 IP                                        Public Ip                                  MAC Address         Group Name                       Multicast IPv4    Multicast IPv6                              Status                       PMTU 
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
10.10.30.4                                 N/A                                        58bf.eacc.0c0b      a0_fabA                       0.0.0.0           ::                                          N/A                          N/A

```
```
ewlc4# show wireless mobility summary 
Mobility Summary

Wireless Management VLAN: 10
Wireless Management IP Address: 10.10.38.4
Wireless Management IPv6 Address: 
Mobility Control Message DSCP Value: 48
Mobility Keepalive Interval/Count: 10/3
Mobility Group Name: a3_fabB
Mobility Multicast Ipv4 address: 0.0.0.0
Mobility Multicast Ipv6 address: ::
Mobility MAC Address: 58bf.ea29.a70b
Mobility Domain Identifier: 0xd2fd

Controllers configured in the Mobility Domain:

 IP                                        Public Ip                                  MAC Address         Group Name                       Multicast IPv4    Multicast IPv6                              Status                       PMTU 
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
10.10.38.4                                 N/A                                        58bf.ea29.a70b      a3_fabB                       0.0.0.0           ::                                          N/A                          N/A

```

### PR DESCRIPTION
**Pull request by robschn**

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_wireless.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        # Mobility Domain Identifier: 0x61b3
+
+        # Controllers configured in the Mobility Domain:
+
+        # IP                                        Public Ip                                  MAC Address         Group Name                       Multicast IPv4    Multicast IPv6                              Status                       PMTU
+        # --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+        # 10.10.7.177                               N/A                                        58bf.ea35.b60b      b80-mobility                0.0.0.0           ::                                          N/A                          N/A
+
+        # Mobility Summary
+        mobility_summary_capture = (
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/show_wireless.py:317]: Do you feel this is easily readable?

Comment (robschn) [reply src/genie/libs/parser/iosxe/show_wireless.py:76]: No I do not, I thought we were supposed to comment the raw output for reference. I can reformat it with just single lines

Comment (robschn) [reply src/genie/libs/parser/iosxe/show_wireless.py:317]: I do! PR submitted!

**Comment by robschn** [comment]
PR submitted: https://github.com/CiscoTestAutomation/genieparser/pull/248

**Pull request merged**